Dockerfile

```diff
@@ -1,9 +1,30 @@
+
+# Stage 0: Build image
+FROM maven:3.8.3-jdk-8
+
+ARG BACKEND_ROOT="WIPP-backend"
+ARG BACKEND_APP="wipp-backend-application"
+ARG BACKEND_DATA="wipp-backend-data"
+ARG BACKEND_CORE="wipp-backend-core"
+ARG BACKEND_ARGO="wipp-backend-argo-workflows"
+
+COPY pom.xml /usr/local/${BACKEND_ROOT}/pom.xml
```

Comment:
Would it make sense to define a `WORKDIR` (`/opt` for example) instead of copying folders to `/usr/local/`?